server/src/server.ts

```diff
@@ -60,6 +61,29 @@ connection.onInitialize(
     },
 );
 
+connection.onHover((_params) => {
+    const doc: TextDocument = documents.get(_params.textDocument.uri);
+    const entityName = getTokenAtPosInDoc(doc.getText(), doc.offsetAt(_params.position));
+    if (!entityName) {
+        return null;
+    }
+    console.warn(`search for ${entityName}`);
```

Comment:
I'm confused by the fact that this is a warning. :thinking: Isn't this something perfectly normal?

Reply:
Oops. I just didn't remove it back then.